docs/UserGuide.md

```diff
@@ -539,50 +472,167 @@ If a parameter is expected only once in the command, but you specified it multip
 **Examples**:
 * Add a meeting and there are no clashes.
     * `meet 1 20.05.2021 15:00 16:00 MRT`
-
       ![meet-add](images/meet-add.png)
-
+      <br><br>
 * Add a meeting but there are clashes.
     * `meet 3 -add 20.05.2021 15:30 17:30 MRT`
-
```

Comment:
Every image from here and the above commands have no empty line between its bullet and the image. Every image after this image has an empty line between the bullet and the image. Perhaps it would be neater if we include the line?